src/components/tables/achievements-table.tsx

```diff
@@ -7,34 +7,126 @@ import {
   TableRow,
   Typography,
 } from "@mui/material";
-
-import { type LoggedUserProfileData } from "~/types/index.js";
+import {
```

Comment:
You can declare multiple type imports like:
`import type { AchievementId, LoggeUserProfileData } from .....`